test/Microsoft.Process.Environment.Tests/EnvironmentManagerWin32Tests.cpp

```diff
@@ -0,0 +1,32 @@
+﻿#include <pch.h>
+#include <EnvironmentManagerWin32Tests.h>
+
+using namespace winrt::Microsoft::ProjectReunion;
+
+namespace ProjectReunionCppTest
+{
+    bool EnvironmentManagerWin32Tests::TestSetup()
+    {
+        winrt::init_apartment(winrt::apartment_type::single_threaded);
```

Comment:
It's TAEF so you can do this via `TEST_CLASS_PROPERTY(L"ThreadingModel", L"STA")` instead of code inside the function

And why STA? Are your tests doing GUI or something that can't do MTA?

And of course...you don't do anything else in TestSetup so why have it? Or is this a placeholder for uipcoming additions?

Reply:
That is an awesome idea.  Lemme change it.

To answer your question of "Why" it is because this is the example I saw for reg-free WinRT.  Even though I saw examples of setting the Threading model in the TEST_CLASS_PROPERTY I didn't connect the two.

And no, this is not a place holder for something in the future.